tests/unit/spec/base/mozilla-traffic-cop.js

```diff
@@ -118,6 +118,54 @@ describe('mozilla-traffic-cop.js', function () {
 
             expect(cop.verifyConfig()).toBeFalsy();
         });
+
+        it('should return true with a valid cookieExpires value', function() {
+            var cop = new Mozilla.TrafficCop({
+                id: 'test123',
+                cookieExpires: 48,
+                variations: {
+                    'v=1': 20,
+                    'v=2': 40
+                }
+            });
+
+            expect(cop.verifyConfig()).toBeTruthy();
+        });
+
+        it('should return false with an invalid cookieExpires value', function() {
+            var cop = new Mozilla.TrafficCop({
+                id: 'test123',
+                cookieExpires: '2 days',
+                variations: {
+                    'v=1': 20,
+                    'v=2': 40
+                }
+            });
+
+            expect(cop.verifyConfig()).toBeFalsy();
+        });
+    });
+
+    describe('Mozilla.TrafficCop.cookieExpiresDate', function() {
+        var config = {
+            id: 'test123',
+            variations: {
+                'v=1': 40,
+                'v=2': 30
+            }
+        };
+
+        it('should return null when no hours are specified', function() {
+            var cop = new Mozilla.TrafficCop(config);
+            expect(cop.cookieExpiresDate()).toBeFalsy();
+        });
+
+        it('should return a date when hours are specified', function() {
+            config.cookieExpires = 72;
+            var cop = new Mozilla.TrafficCop(config);
+            expect(cop.cookieExpiresDate()).toBeTruthy();
+            expect(Object.prototype.toString.call(cop.cookieExpiresDate())).toEqual('[object Date]');
```

Comment:
If we made it possible to pass in an optional `date` parameter to `cookieExpiresDate()` for testing, would this make it easier to assert that the returned value is what we expect it to be given the value in `config`?